datasource/opennms-kafka/pom.xml

```diff
@@ -22,6 +22,7 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
                         <Import-Package>
+                            !org.apache.kafka.streams.KafkaStreams,
```

Comment:
This one necessary? If so, can we add a comment to explain why it's needed.

Reply:
We need it because otherwise the Maven Bundle plugin treats KafkaStreams as package even though its a class. It thinks its a package since we reference the inner class KafkaStreams.State. I'll add a comment.